assets/rpip-10/expense_3.md

```diff
@@ -0,0 +1,23 @@
+This expense funds the protocol development team for 12 months.
+
+The amount will be paid in a lump-sum payment of: `53,172.97 RPL`.
+
+### Breakdown
+
+| Period      | Period Date |Inflation Total | Development Funding |
+| ----------- | ----------- |----------- | ----------- |
+| 15          | 26/10/2023  | 74,126.81   | 3,706.36  | 
```

Comment:
How were these "inflation total" numbers calculated?

Going off of inflation events doesn't match <https://dune.com/queries/1658975>. Period 16 matches <https://etherscan.io/tx/0xe4d485c1620a160f5ad57e9f445b6c756e46a3d917234109a6c4b059fabca6d3>, but Period 15 doesn't <https://etherscan.io/tx/0xbfc588deee477b49c9369a021d67944b7d42144ce71a2dbc3824c76e8144e67c>.

Also doesn't quite match using the pDAO distribution <https://dune.com/queries/1264376> and then dividing by pDAO share to get total inflation.

Iirc, it's expected that those 2 methods don't quite match for some reason. Maybe like we end up transferring n-1 days from the current period and 1 day from the last.

Anyhow -- a clear way to replicate would be great.

Reply:
The values were created using the "Inflation Changes" spreadsheet. You are right, for some reason period 15 was incorrect. I have adjusted the spreadsheet and cross checked based on the inflation events / RPL supply. 

There is a choice to make though. Do we use the inflation event number or the reward file inflation number? Generally we use the reward file inflation number because it is more accurate. When the inflation event is fired it mints all inflation up until that point so it is point in time. There is a very consistent difference between the inflation event number and the reward file inflation number (although more than I expect...).

I have calculated the factor in the "Inflation Changes" spreadsheet based on the last  5 periods and use that to predict the reward file number. Although I'd like to see the next reward period to make sure the projection works.

Either way I think it works out in the end. Thoughts?